service/src/main/java/greencity/security/service/OwnSecurityServiceImpl.java

```diff
@@ -184,7 +196,10 @@ public SuccessSignUpDto signUpEmployee(EmployeeSignUpDto employeeSignUpDto, Stri
         String password = generatePassword();
         employeeSignUpDto.setPassword(password);
         OwnSignUpDto dto = modelMapper.map(employeeSignUpDto, OwnSignUpDto.class);
+
         User employee = createNewRegisteredUser(dto, jwtTool.generateTokenKey(), language);
+        employee.setRole(Role.ROLE_UBS_EMPLOYEE);
```

Comment:
Why do you set `Role.ROLE_UBS_EMPLOYEE` twice?
![image](https://user-images.githubusercontent.com/15122465/226325933-f485338e-d9f4-4797-b496-abc07c7e67df.png)